cmd/system-probe/config/ns.go

```diff
@@ -0,0 +1,40 @@
+// Unless explicitly stated otherwise all files in this repository are licensed
+// under the Apache License Version 2.0.
+// This product includes software developed at Datadog (https://www.datadoghq.com/).
+// Copyright 2016-present Datadog, Inc.
+
+package config
+
+import "strings"
+
+func spNS(k ...string) string {
```

Comment:
I found these abbreviations a bit inscrutable without foreknowledge of what the products were.  If it's not too verbose, it might be helpful to spell them out.

Reply:
The point is to keep them less verbose. I will add comments.